src/background/services/openPayments.ts

```diff
@@ -453,4 +480,66 @@ export class OpenPaymentsService {
       keyId
     })
   }
+
+  async createQuote({
+    walletAddress,
+    receiver,
+    amount
+  }: CreateQuoteParams): Promise<Quote> {
+    return await this.client!.quote.create(
+      {
+        accessToken: this.token.value,
+        url: walletAddress.resourceServer
+      },
+      {
+        method: 'ilp',
+        walletAddress: walletAddress.id,
+        receiver,
+        debitAmount: {
+          value: amount,
+          assetCode: walletAddress.assetCode,
+          assetScale: walletAddress.assetScale
+        }
+      }
+    )
+  }
+
+  async createOutgoingPayment({
+    walletAddress,
+    quoteId
+  }: CreateOutgoingPaymentParams): Promise<OutgoingPayment> {
+    return await this.client!.outgoingPayment.create(
+      {
+        accessToken: this.token.value,
+        url: walletAddress.resourceServer
+      },
+      {
+        quoteId,
+        walletAddress: walletAddress.id,
+        metadata: {
+          source: 'Web Monetization'
+        }
+      }
+    )
+  }
+
+  async rotateToken() {
+    const rotate = this.deduplicator.dedupe(this.client!.token.rotate, {
+      cacheFnArgs: false
+    })
+    const token = await rotate({
+      url: this.token.manage,
+      accessToken: this.token.value
+    })
+    await this.storage.set({
+      token: {
+        value: token.access_token.value,
+        manage: token.access_token.manage
+      }
+    })
+    this.token = {
+      value: token.access_token.value,
+      manage: token.access_token.manage
+    }
```

Comment:
Nit: Let's set `this.token` first and reuse in `this.storage.set` to avoid duplication. If concerned about `storage.set` throwing and `this.token` being inconsistent, then can assign to a new variable first.